.github/workflows/test.yml

```
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4.1.5
       - uses: actions/setup-go@v5
         with:
-          go-version: "1.18"
+          go-version: "1.22"
```

```suggestion
          go-version-file: "testdata/go.mod"
```

How about referring to `testdata/go.mod` for the Go version?

---

Thank you for the suggestion! LGTM